README.md

```diff
@@ -1,7 +1,7 @@
 # SmartDeviceLink
 # RPC Spec
 
-###### Version: 6.0.0
+###### Version: 6.2.0
```

Comment:
@LitvinenkoIra to remove this

Reply:
@LitvinenkoIra reverted in the e6ed719336f78dea817c9fee65fecc4ebda6cdce